test/resources/microshift-process.resource

```diff
@@ -80,17 +80,9 @@ Wait For MicroShift Service
 
 Restart MicroShift
     [Documentation]    Restart the MicroShift service
-    # Use separate stop and start steps to avoid a race condition with
-    # restart where we can't tell when it is down and so we can't tell
-    # whether we should count it as back up yet. Forcing the service
-    # down, then back up, ensures that when we do things like check
-    # systemd settings for the running process we get the values we
-    # expect without worrying about the race. Any test that modifies
-    # the MicroShift configuration file will be more reliable as a
-    # result.
-    Stop MicroShift
-    Sleep    10 seconds
-    Start MicroShift
+    ${cur_pid}=    MicroShift Process ID
```

Comment:
We should also make sure that microshift-etcd is restarting.

Reply:
done